### ISSUE TITLE
Review list of Imports in frameworks section of project.json for Web templates

### ISSUE THREAD

**Issue by rustd** [aspnet/Templates#447]
In the framework section of project.json https://github.com/aspnet/Templates/blob/dev/src/BaseTemplates/StarterWeb/project.json#L28

we have the following. Is this final?

``` html
"frameworks": {
    "netstandardapp1.5": {
      "imports": [
        "dnxcore50",
        "portable-net45+win8"
      ]
    }
  },
```

**Comment by rustd** [comment]
@anurse can you please confirm whether the name of dotnet5.5 is correct? If not what should it be.

**Comment by analogrelay** [comment]
It should be `dotnet5.6` technically, since `dotnet` versions started at `5.1` whereas `netstandard` started at `1.0`

**Comment by phenning** [comment]
Current template does not have dotnetx.x at all: - is this correct?

```
 "netcoreapp1.0": { 
    "imports": [ 
      "dnxcore50", 
      "portable-net45+win8" 
   ] 
}
```

**Comment by analogrelay** [comment]
Yes, that should be fine. Imports are only needed if you need to take a dependency on a package that hasn't been updated to target the right frameworks :). Assuming all the packages that the template uses have been updated, the risk is just if a user adds a different package that hasn't been updated, NuGet will complain that they are incompatible (since we decided NOT to make `netstandard` and `dotnet` compatible, to make a clean break and not create more legacy)

**Comment by rustd** [comment]
Ok. So here is the final imports list for RC2 for Web templates. /cc @glennc @davidfowl 

"frameworks": {
    "netcoreapp1.0": {
      "imports": [
        "dotnet5.6",
        "dnxcore50",
        "portable-net45+win8"
      ]
    }
  },

**Comment by phenning** [comment]
@rustd I don't think that is right, we don't have any package included in the templates that _require_ dotnet5.6 to be specified. Shouldn't we be including the minimal set of imports for each template based on the packages the template pulls in by default?

**Comment by rustd** [comment]
We are including a more broader set of imports

**Comment by phenning** [comment]
Fixed: a44873b86f0ecf8bc4f6547900fc2c6f2db52b62
